reckless_lib/src/plugin.rs

```diff
@@ -16,12 +17,24 @@ pub enum PluginLang {
 }
 
 impl PluginLang {
-    pub fn default_install(&self) -> Result<String, RecklessError> {
+    pub async fn default_install(&self, path: &str, name: &str) -> Result<String, RecklessError> {
         match self {
             PluginLang::Python => {
                 /* 1. RUN PIP install or poetry install
                  * 2. return the path of the main file */
-                todo!()
+                let req_file = format!("{}/requirements.txt", path);
+                let main_file = format!("{}/{}.py", path, name);
+                match Command::new("pip")
+                    .arg("install")
+                    .arg(req_file.as_str())
```

Comment:
```suggestion
                    .arg("install")
                    .arg("-r")
                    .arg(req_file.as_str())
```

it is missing the `-r`?